blueprints/ember-cli-blueprint-test-helpers/index.js

```diff
@@ -1,6 +1,6 @@
-var fs = require('fs-extra');
-var path = require('path');
-var EOL = require('os').EOL;
+const fs = require('fs-extra');
```

Comment:
The files that use const/let need `'use strict';` at the top of the file to work.